packages/framework/presence/src/test/presenceManager.spec.ts

```diff
@@ -429,6 +501,131 @@ describe("Presence", () => {
 								clientToDisconnect,
 							);
 						});
+
+						it.skip("updates stale attendee status to 'Disconnected' after 30s delay upon reconnection", () => {
+							assert(knownAttendee !== undefined, "No attendee was set in beforeEach");
+
+							// Act - disconnect & reconnect local client
+							runtime.disconnect(); // Simulate local client disconnect
+							clock.tick(1000);
+							runtime.connect("client6"); // Simulate local client reconnect with new connection id
+
+							// Verify - stale attendee should still be 'Connected' after 15 seconds
+							clock.tick(15_001);
+							assert.strictEqual(
+								knownAttendee.getConnectionStatus(),
+								SessionClientStatus.Connected,
+								"Stale attendee should still be 'Connected' after 15s",
+							);
+
+							// Verify - stale attendee should be 'Disconnected' after 30 seconds and announced via `attendeeDisconnected`
+							clock.tick(15_001);
+							assert.strictEqual(
+								knownAttendee.getConnectionStatus(),
+								SessionClientStatus.Disconnected,
+								"Stale attendee should be 'Disconnected' 30s after reconnection",
+							);
+							assert.strictEqual(
+								disconnectedAttendees.length,
+								1,
+								"Exactly one attendee should be announced as disconnected",
+							);
+						});
+
+						it.skip("does not update stale attendee status if local client does not reconnect", () => {
+							assert(knownAttendee !== undefined, "No attendee was set in beforeEach");
+
+							// Act - disconnect local client
+							runtime.disconnect();
+
+							// Verify - stale attendee should still be 'Connected' after 30 seconds
+							clock.tick(30_001);
```

Comment:
the tick is part of the action
you can use a larger number (a much larger number)
in the other cases the time past is 31,002.

Reply:
Changed to 60 seconds here and moved tick under action